pkg/controllers/rollout/controller.go

```diff
@@ -364,25 +364,7 @@ func (r *Reconciler) updateBindings(ctx context.Context, latestResourceSnapshotN
 // It reconciles on the CRP when a new resource resourceBinding is created or an existing resource binding is created/updated.
 func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 	r.recorder = mgr.GetEventRecorderFor("rollout-controller")
-	// TODO: consider converting this to a customized controller with a queue
-	return ctrl.NewControllerManagedBy(mgr).
-		// This is a hack, we don't actually watch the CRP but the controller runtime requires us to specify a for type
-		For(&fleetv1beta1.ClusterResourcePlacement{}, builder.WithPredicates(
-			&predicate.Funcs{
-				CreateFunc: func(createEvent event.CreateEvent) bool {
-					return false
-				},
-				UpdateFunc: func(e event.UpdateEvent) bool {
-					return false
-				},
-				DeleteFunc: func(deleteEvent event.DeleteEvent) bool {
-					return false
-				},
-				GenericFunc: func(genericEvent event.GenericEvent) bool {
-					return false
-				},
-			},
-		)).
+	return ctrl.NewControllerManagedBy(mgr).Named("rollout_controller").
```

Comment:
wow, we don't need this any more. that's awesome!

Reply:
yeah, that's the reason I wanted to pick up this version. I forgot where I saw a bug/issue that they want to remove the requirement for "For" so I was like yeah, I want this.